frontend/packages/dev-console/src/utils/pipeline-filter-reducer.ts

```diff
@@ -12,25 +12,30 @@ export const pipelineFilterReducer = (pipeline): string => {
   return pipeline.latestRun.status.succeededCondition;
 };
 
-export const pipelineRunFilterReducer = (pipelineRun): string => {
+export const pipelineRunStatus = (pipelineRun): string => {
```

Comment:
```suggestion
export const pipelineRunStatus = (pipelineRun: PipelineRun): string => {
```

Reply:
Yeah, circular reference unfortunately... wasn't sure I wanted to refactor where the types live in this PR.

Reply:
Ah, that's unfortunate. OK.